protos/processor.proto

```diff
@@ -50,6 +50,32 @@ message TpRegisterResponse {
     Status status = 1;
 }
 
+// The unregistration request from the transaction processor to the validator/executor
+message TpUnRegisterRequest {
```

Comment:
I like the approach of sending less and sending just one unregister message per request to disconnect even if there are multiple handlers. But now I can't think of anything that would need to be in the message since the zeromq identity is already known on the validator side.